src/components/picture/_children/source.jsx

```diff
@@ -0,0 +1,41 @@
+import PropTypes from "prop-types";
+import formatSrcset from "../../../utils/format-srcset";
+
+const Source = ({ height, media, resizedOptions, resizerURL, src, width, ...rest }) => {
+	if (resizedOptions?.auth) {
+		return (
+			<source
+				{...rest}
+				height={height}
+				media={media}
+				srcSet={formatSrcset(resizerURL.concat(src), resizedOptions, width, height)}
+				width={width}
+			/>
+		);
+	}
+	return <source {...rest} media={media} srcSet={src} />;
```

Comment:
Do we need `width` and `height` attributes here?

Reply:
@matthewroach :Thank you, great catch! Added both height and width.